package.json

```diff
@@ -113,7 +113,7 @@
     "turbo": "^2.1.2",
     "typescript": "~5.6.2"
   },
-  "packageManager": "npm@8.19.4",
+  "packageManager": "npm@8.19.4+sha512.dc700d97c8bd0ca9d403cf4fe0a12054d376f048d27830a6bc4a9bcce02ec42143cdd059ce3525f7dce09c6a4e52e9af5b996f268d8729c8ebb1cfad7f2bf51f",
```

Comment:
Can you explain how to find the hash for this?
So that if I want to update it some point I know how to keep this security improvement.

Do you also have more information/resources on why this is a good idea?

Reply:
I wrote it manually originally, but remembered a few days ago (in a personal project) that there's a cli, so getting the hash is as simple as `corepack use npm@x.x.x`, and it'll add the hash in automatically.

[Corepack then does an integrity check when downloading](https://github.com/nodejs/corepack?tab=readme-ov-file#when-authoring-packages:~:text=The%20hash%20is%20optional%20but%20strongly%20recommended%20as%20a%20security%20practice.)